lib/workers/repository/dependency-dashboard.ts

```diff
@@ -220,15 +241,36 @@ function getBranchesListMd(
   if (filteredBranches.length === 0) {
     return '';
   }
-  let result = `## ${title}\n\n${description}\n\n`;
-  result += `${filteredBranches
-    .map((branch: BranchConfig): string => getListItem(branch, listItemType))
-    .join('')}`;
+
+  const { categories, uncategorized } =
+    splitBranchesByCategory(filteredBranches);
+  let result = `## ${title}\n\n${description}`;
+  if (Object.keys(categories).length > 0) {
```

Comment:
store `Object.keys(categories).length` in a variable, it's used / computed more than once.